Lib/glyphsLib/builder/context.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import (print_function, division, absolute_import,
+                        unicode_literals)
+
+from collections import OrderedDict
+
+
+class GlyphsToUFOContext(object):
+    """Builder context for Glyphs to UFO + designspace."""
+
+    def __init__(self, font, defcon):
+        """Create a context for the Glyphs to UFO + designspace builder.
+
+        Keyword arguments:
+        font -- The GSFont object to transform into UFOs
+        defcon -- The defcon module to use to build UFO objects (you can pass
+                  a custom module that has the same classes as the official
+                  defcon to get instances of your own classes)
+        """
+        self.font = font
+        self.defcon = defcon
+
+        self.ufos = OrderedDict()
+        """
```

Comment:
any reasons why here you use `""" multi-line string literals """` instead of simple `# inline comment`?

Reply:
Because I read in a PEP that it was a way to document class attributes:

> String literals occurring immediately after a simple assignment at the top level of a module, class, or __init__ method are called "attribute docstrings".

Is there another way?

Reply:
Interesting, I didn't know that...
However, if you try to do `help(...)` on that class, it only shows the first multi-line string.

Reply:
maybe auto docs generators like sphinx will be able to retrieve those. I'm fine if we leave them. We should polish the documentation at some point.